src/settings/ThemeScreen.js

```diff
@@ -0,0 +1,89 @@
+/* @flow strict-local */
+
+import React from 'react';
+import { View, FlatList, Text } from 'react-native';
+
+import type { RouteProp } from '../react-navigation';
+import { connect } from '../react-redux';
+import { getSettings } from '../selectors';
+import '../boot/AppEventHandlers';
+import type { AppNavigationProp } from '../nav/AppNavigator';
+import { settingsChange } from '../actions';
+import type { Dispatch } from '../types';
+import { Screen, Touchable, OptionDivider } from '../common';
+import { IconDone } from '../common/Icons';
+import { createStyleSheet, BRAND_COLOR, ThemeContext } from '../styles';
+
+const styles = createStyleSheet({
+  listWrapper: {
+    flex: 1,
+    flexDirection: 'column',
+  },
+  name: {
+    fontWeight: '300',
+    fontSize: 13,
+  },
+  listItem: {
+    flexDirection: 'row',
+    alignItems: 'center',
+    paddingTop: 16,
```

Comment:
It looks like these styles were borrowed from `LanguagePickerItem`, but with `paddingTop` and `paddingBottom` set to 12 instead of 16. Can you talk about the choice for what goes in `styles`?

Reply:
Ahh, I think I missed the changes I made to the styles. I am now using the same styles as mentioned in the `LanguagePickerItem` i.e making the `paddingTop` and `paddingBottom` set to 12 again.